packages/apps/reputation-oracle/server/src/modules/kyc/kyc.service.spec.ts

```diff
@@ -170,44 +145,37 @@ describe('Kyc Service', () => {
 
     it('Should start a Kyc session for the user', async () => {
       const mockUserEntity = {
-        kyc: {
-          sessionId: null,
-          url: null,
-        },
-        id: 1,
-        email: 'test@example.com',
+        id: faker.number.int(),
       };
 
-      httpService.post = jest.fn().mockImplementation(() => {
-        return of({
-          data: {
-            status: 'success',
-            verification: {
-              id: 123,
-              url: 'https://randomurl.test',
-            },
+      const mockPostKycRespose = {
```

Comment:
You can use `createHttpServiceResponse` to create responses with proper structure and types